pkg/vpcmodel/explainabilityInput.go

```diff
@@ -28,6 +27,100 @@ func (e *ExplanationArgs) Dst() string {
 	return e.dst
 }
 
+// consts for managing errors from the single vpc context in the global, multi-vpc, context.
+const (
+	noErr                        = iota
+	fatalErr                     // fatal error that implies immediate termination (do not wait until we go over all vpcs)
+	internalNotWithinSubnetsAddr // internal address with not within vpc config's subnet addr - wait until we go over all vpcs
+	noValidInputErr              // string does not represent a valid input w.r.t. this config - wait as above
+)
+
+const (
+	noInternalIP        = iota // nor src neither dst were given as internal IP
+	srcInternalIP              // only src was given as internal ip
+	dstInternalIP              // only dst was given as internal ip
+	srcAndDstInternalIP        // both src and dst given as internal ip
+)
+
+const noValidInputMsg = "does not represent a legal IP address, a legal CIDR or a VSI name"
+
+// getVPCConfigAndSrcDstNodes given src, dst names returns the config in which the exaplainability analysis of these
+// should be done and the Nodes for src and dst.
+
```

Comment:
```suggestion
```

Reply:
why? I've updated the above description to include all outputs and kept it.

Reply:
This is just a deletion of empty line.
The lines that document a certain function should be consecutive, with no empty lines.